code/API_definitions/qod-api.yaml

```diff
@@ -115,6 +115,54 @@ paths:
                 message: "Session could not be created"
         "503":
           $ref: "#/components/responses/Generic503"
+    get:
+      tags:
+        - QoS Sessions
+      summary: Fetch all paginated sessions created
+      description: Fetch QoS sessions created
+      operationId: listSessions
+      parameters:
+        - in: query
+          name: page
+          schema:
+            type: interger
+          required: true
+        - in: query
+          name: pageSize
```

Comment:
```suggestion
          name: size
```

Reply:
I have camel-case in parameters, so I tend to keep them shorter. It's a pity that after all these years the dev community still hasn't found a standard way to do pagination...
Some kind of "authority" at least in the Java world is spring - and there, the typical parameter names are `page`, `size` and `sort`, see https://www.baeldung.com/rest-api-pagination-in-spring - so that's where my suggestion came from.